pallets/automation-time/src/lib.rs

```diff
@@ -1184,23 +1186,6 @@ pub mod pallet {
 			(task, <T as Config>::WeightInfo::run_auto_compound_delegated_stake_task())
 		}
 
-		fn compound_delegator_stake(
```

Comment:
For my sanity, this is unrelated to v0.9.26 upgrade and is just a parachain-staking refactor that came with the dependency upgrade?

Reply:
Yes and no. The `v0.9.26` upgrade for parachain-staking included some large functional changes around funds being locked vs reserved. Rather than expose the implementation details of the locked funds outside parachain staking, I chose to move the "delegator bonds up to a minimum amount" functionality into parachain-staking.